### PR TITLE
Adding repo files for sources and topological_inventory stable releases

### PR DESCRIPTION
**Pull request by gtanzillo**

### CODE REVIEW

File: config/topological_inventory.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+stable:
```
Comment (Fryguy) [inline config/topological_inventory.yml]: While the code says "branch", that's a poor naming choice on my side.  Think of it more as a "repo_set".  So in that way you need a unique name.  In this case, this stable will collide with the sources stable.

I think here, you could use `topological-stable` perhaps (and `sources-stable`)

File: config/topological_inventory.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  inventory_refresh:
```
Comment (Fryguy) [inline config/topological_inventory.yml]: This is a real released gem, so I don't think you need it here.

File: config/sources.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+stable:
+  sources-api:
+  manageiq-api-common:
+
```
Comment (Fryguy) [inline config/sources.yml]: extra line?

File: config/sources.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+stable:
+  sources-api:
+  manageiq-api-common:
```
Comment (Fryguy) [inline config/sources.yml]: We are going to make a gem out of this, so I think we should remove it and release it.

**Comment by Fryguy** [comment]
As discussed, you can have multiple repo sets for different purposes, so perhaps a `topological-stable` for release purposes, and a `topological-all` for when you need to blast out stuff.  (of course add the latter only when you need it)

**Comment by Fryguy** [comment]
Also, file names should be `repo.sources.yml` and `repo.topological.yml`

**Comment by djberg96** [comment]
👍 Will re-run the blaster for codeclimate once this is ready.

**Comment by Fryguy** [comment]
cc @gmcculloug Can you make ones like this for catalog/approval?

**Pull request merged**